src/main/java/com/dougnoel/sentinel/steps/BaseSteps.java

```diff
@@ -172,4 +175,10 @@ public static void pressBrowserButton(String option) {
         }
     }
     
-}
\ No newline at end of file
+    @Then("^video recording (.*?)( was not)?(?: was)? attached to the report?$")
+    public static void addVideoToReport(String elementName, String assertion) {
+    	if(Configuration.toString("addVideoToReport")!=null && !elementName.contains("not"))
```

Comment:
If the step exists, you shouldn't have to turn it on in the commandline. Ideally this would happen whether there was a cucumber step or not by adding it to the after step for each method.
```suggestion
    	if(Configuration.toString("addVideoToReport")!=null && !assertion.contains("not"))
```